handler/endpoint.go

```diff
@@ -169,7 +169,8 @@ func (e *Endpoint) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 
 	// always apply before write: redirect, response
 	if err = eval.ApplyResponseContext(evalContext, e.opts.Context, clientres); err != nil {
-		log.WithError(err).Error()
+		e.opts.Error.ServeError(err).ServeHTTP(rw, req)
```

Comment:
This change breaks adding a modifier to the response writer (headers from server or api context).

Reply:
And I am wondering why this is not tested.

Reply:
Why this was changed?